scripts/download_backtest_data.py

```diff
@@ -127,7 +127,7 @@
 
             logger.info(f'Downloading pair {pair}, interval {ticker_interval}.')
             download_pair_history(datadir=dl_path, exchange=exchange,
-                                  pair=pair, ticker_interval=ticker_interval,
+                                  pair=pair, ticker_interval=str(ticker_interval),
```

Comment:
Is not it a string always? ('1h', '5m'?)

Reply:
technically yes - but mypy does not know as args.ticker_interval returns `Optional[Any]` ... not List[str] ...